packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -238,6 +247,10 @@ MonthPicker.propTypes = {
    * Additional hint text to display
    */
   hint: PropTypes.node,
+  /**
+   * Heading type to override default `<h4>` in title block
+   */
+  headingType: PropTypes.string,
```

Comment:
Nit: I think `headingLevel` and `PropTypes.number` is more accurate for the value we're expecting here.

However, do we need to provide an option for setting the heading level?

Reply:
Yes, we need a way to set the heading level. This component could be places where peer headings are not `<h4>`'s, so having that override is important for semantic reasons. Screen reader users often navigate by skimming headings; I'd like the option to match peer headings through prop override when necessary.

Reply:
A number makes more sense, so I'll update that.

Reply:
@1Copenut Thoughts on rephrasing to `headingLevel`? This is the typical vocabulary when referring to the heading options